packages/web/src/components/banner/TanQueryFeedbackBanner.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react'
+
+import { CallToActionBanner } from './CallToActionBanner'
+
+const messages = {
+  text: 'Leave us some feedback',
```

Comment:
should this mention that it's about tan-query?

maybe `'Leave us some feedback about this test version of the app'`